hummingbird/static/js/hummingbird-ajax.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function() {
+
+$('.delete-user').hide();
```

Comment:
nit: I would add an indent to the whole file (except for the first and last lines) to show this is all under `$(document).ready()`.